examples/complete/main.tf

```diff
@@ -28,7 +26,7 @@ module "subnets" {
 }
 
 module "codefresh_backing_services" {
-  source          = "git::https://github.com/cloudposse/terraform-aws-codefresh-backing-services.git?ref=0.1.0"
```

Comment:
restore to original?

Reply:
`example` easy to run based on the local directory.